### PR TITLE
Update README to current dev environment

### LINKED ISSUE #2112: Update dev docs to reflect that building logstash requires Ruby 1.9
While trying to build LogStash on a RHEL system:
     $ rake bootstrap
     rake aborted!
     NoMethodError: undefined method `require_relative' for main:Object
     (path redacted)/logstash/rakelib/plugin.rake:1
    (See full trace by running task with --trace)
After asking about this on IRC, it came to light that require_relative is a new feature in Ruby 1.9, while my system has Ruby 1.8.7. I would expect this requirement to be documented at https://github.com/elasticsearch/logstash#developing.

### PR DESCRIPTION
**Pull request by suyograo**
Fixes #2112, #1527

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,8 +63,15 @@ Notes about using other rubies. If you don't use rvm, you can probably skip
 this paragraph. Logstash works with other rubies, and if you wish to use your
 own ruby you must set `USE_RUBY=1` in your environment.
 
-We recommend using flatland/drip for faster startup times during development. To
-tell Logstash to use drip, set `USE_DRIP=1` in your environment.
+## Drip Launcher
+
+[Drip](https://github.com/ninjudd/drip) is a launcher for the Java Virtual Machine that provides much faster startup times than the `java` command. The drip script is intended to be a drop-in replacement for the java command, only faster. We recommend using drip during development. 
+
+To tell Logstash to use drip, either set the `USE_DRIP=1` environment variable or set `JAVACMD=which drip`.
```
Comment (talevy) [inline README.md]: the javacmd should be set like

```
JAVACMD=`which drip`
```

I am not sure how to escape the back-tick in markdown

Comment (suyograo) [inline README.md]: right, checking md syntax

**Comment by suyograo** [comment]
@talevy done, fixed the backtick rendering

**Comment by talevy** [comment]
doubletick!

LGTM

**Pull request closed without merging**